cmd/necogcp/cmd/createrunner.go

```diff
@@ -0,0 +1,195 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/cybozu-go/log"
+	"github.com/cybozu-go/neco-gcp/pkg/gcp"
+	"github.com/cybozu-go/neco-gcp/pkg/github"
+	"github.com/cybozu-go/well"
+	"github.com/spf13/cobra"
+	"google.golang.org/api/googleapi"
+)
+
+const (
+	SERVICE_ACCOUNT = "bootstrap-dctest@neco-test.iam.gserviceaccount.com"
+)
+
+var (
+	appId          int64
+	installationId int64
+	privateKeyPath string
+	pat            string
+	runnerRepo     string
+	runnerName     string
+	labels         []string
+)
+
+var runnerStartUpScriptTmpl = template.Must(template.New("").Parse(`#!/bin/sh
+
+useradd -m runner
+usermod -aG google-sudoers runner
+RUNNER_HOME=/home/runner
+/snap/bin/gcloud secrets versions access latest --secret="ghatoken_{{ .RunnerName }}" > $RUNNER_HOME/github_token &&
+cat << EOF >job-cancelled
+#!/bin/sh
+EOF
+chmod 755 job-cancelled
+cp job-cancelled /usr/local/bin/job-started
+cp job-cancelled /usr/local/bin/job-failure
+cp job-cancelled /usr/local/bin/job-success
+mv job-cancelled /usr/local/bin/job-cancelled
+
+NUM_DEVICES=$(ls /dev/nvme0n*|wc -l)
+mdadm --create /dev/md0 -l stripe --raid-devices=${NUM_DEVICES} /dev/nvme0n*
+mkfs -t ext4 -F /dev/md0
+mkdir -p /var/scratch
+mount -t ext4 /dev/md0 /var/scratch
+chmod 1777 /var/scratch
+
+curl -L -o $RUNNER_HOME/runner.tar.gz https://github.com/actions/runner/releases/download/v2.298.2/actions-runner-linux-x64-2.298.2.tar.gz
+tar xzf $RUNNER_HOME/runner.tar.gz -C $RUNNER_HOME
+$RUNNER_HOME/bin/installdependencies.sh
+
+cat << 'EOT' >$RUNNER_HOME/run-actions.sh
+#!/bin/sh
+REPO_URL=https://github.com/{{ .RunnerRepo }}
+
+export GOPATH=${HOME}/go
+export GOBIN=${GOPATH}/bin
+export PATH=${GOBIN}:/usr/local/go/bin:${PATH}
+export NECO_DIR=${GOPATH}/src/github.com/cybozu-go/neco
+export NECO_APPS_DIR=${GOPATH}/src/github.com/cybozu-private/neco-apps
+
+$HOME/config.sh \
+	--unattended \
+	--replace \
+	--name $(hostname) \
+	--labels gcp-runner {{ .LabelOption }} \
```

Comment:
```suggestion
	--labels gcp-runner{{ .LabelOption }} \
```
If there is a space here, it is likely to cause unintended movement.

However, if you include the gcp-runner label in the labelOption variable, I don't see a problem.
https://github.com/cybozu-go/neco-gcp/pull/127/files#r1035815939